pkg/daemon/ceph/client/pool.go

```diff
@@ -183,11 +183,24 @@ func CreatePool(context *clusterd.Context, clusterInfo *ClusterInfo, clusterSpec
 }
 
 func CreatePoolWithPGs(context *clusterd.Context, clusterInfo *ClusterInfo, clusterSpec *cephv1.ClusterSpec, pool cephv1.NamedPoolSpec, appName, pgCount string) error {
+	autoscaleMode := pool.AutoscaleMode
 	if pool.Name == "" {
 		return errors.New("pool name must be specified")
 	}
+	if pool.Parameters != nil {
+		placementGroupCount, ok := pool.Parameters["pg_num"]
+		if ok {
+			pgCount = placementGroupCount
+			if autoscaleMode == cephv1.PoolAutoscaleModeOn {
+				logger.Warning("The PGs will be managed by the Ceph PG autoscaler, because of the autoscale mode is on, the settings of the PGs is invalid")
+			} else if autoscaleMode == "" {
+				// If 'pg_num' has set, the default value of the autoscale mode will be off.
+				autoscaleMode = cephv1.PoolAutoscaleModeOff
```

Comment:
I think it would be good to add a `logger.Debugf` message here in lieu of the comment.